### DISCUSSION TITLE
Remove "Send a test message" from bar

### DISCUSSION THREAD

**Discussion by tonnic** [berrystephenw/OctoText#23] [Ideas]
Can you please remove the "Send a test message" from the bar, or at least toggle it on or off

**Comment by berrystephenw** [comment]
I'll put it on the list. Thanks!

**Reply by scorfman** [reply]
Seconded. Handy at times so making it optional is my vote.

**Comment by berrystephenw** [comment]
How about this for an icon when it's visible:
![image](https://user-images.githubusercontent.com/44931130/113442988-1547dd80-93bf-11eb-8c9c-4b7fabc5d98a.png)

**Reply by garyriet** [reply]
Thats great

**Reply by scorfman** [reply]
+1

**Comment by tonnic** [comment]
Definitely less intrusive, but would still be nice to be able to hide it if wanted.

**Reply by berrystephenw** [reply]
yup, one step at a time ;)

**Comment by berrystephenw** [comment]
anyone interested in this feature can switch to the rc branch, see #34 and I've implemented this change with a disappearing icon.
